README.md

```diff
@@ -1,14 +1,9 @@
 # cargo-semver-checks-action
 Lint your crate API changes for semver violations.
```

Comment:
```suggestion
Lint your crate API changes for semver violations.

*This is the work-in-progress v2 version of this action. Find the docs for the stable v1 action [here](https://github.com/obi1kenobi/cargo-semver-checks-action/tree/v1.4).*
```
Since the action isn't yet complete and you're still planning on adding major pieces of functionality.